docs/guides/creating-interactive-workflows.md

```diff
@@ -162,3 +213,200 @@ def get_shirt_order():
+
+### Sending input
+
+You can send input to a flow with the `send_input` function. This works similarly to `receive_input`, taking the same types for the `run_input` argument (basic types like `str`, `BaseModel` subclasses, and `RunInput` subclasses).
+
+!!! note "When can you send input to a flow run?"
+    You can send input to a flow run as soon as you have the flow run's ID. The flow does not have to be receiving input for you to send input. If you send a flow input before it is receiving, it will see your input when it calls `receive_input` (as long as the types in the `send_input` and `receive_input` calls match!)
+
+Next, we'll create a `sender` flow that starts a `greeter` flow run and then enters a loop, continuously getting input from the terminal and sending it to the greeter flow:
+
+```python
+@flow
+async def sender():
+    greeter_flow_run = await run_deployment(
+        "greeter/send-receive", timeout=0, as_subflow=False
+    )
+
+    greetings_seen = set()
+
+    while True:
+        name = input("What is your name? ")
+        if not name:
+            continue
+
+
+        if name == "q" or name == "quit":
+            await send_input(EXIT_SIGNAL, flow_run_id=greeter_flow_run.id)
+            print("Goodbye!")
+            break
+
+        await send_input(name, flow_run_id=greeter_flow_run.id)
+
+        async for greeting in receive_input(
+            str, with_metadata=True, exclude_keys=greetings_seen, timeout=None, poll_interval=0.1
+        ):
+            print(greeting)
+            greetings_seen.add(greeting.metadata.key)
+            break
+```
+
+There's more going on here than in `greeter`, so let's take a closer look at the pieces.
+
+First, we use `run_deployment` to start a `greeter` flow run. This means we must have a worker or `flow.serve()` running in separate process. That process will begin running `greeter` while `sender` continues to execute. Calling `run_deployment(..., timeout=0)` ensures that `sender` won't wait for the `greeter` flow run to complete, because, of course, it's running a loop and will only exit when we send `EXIT_SIGNAL`.
+
+Next, what's going on with `greetings_seen`? This flow works by entering a loop, and on each iteration of the loop, the flow asks for terminal input, sends that to the `greeter` flow, and then runs *another loop* when it calls `async for greeting in receive_input(...)`. As we saw earlier in this guide, `receive_input` always starts at the beginning of all input this flow run received. Keeping track of the inputs we've already seen in the `greetings_seen` set allows us to pass this set in for the `exclude_keys` parameter each time we reenter the loop and call `receive_input` again.
+
+Next, we let the terminal user who ran this flow exit by entering the string "q" or "quit," and when that happens, we also seend the `greeter` flow an exit signal so it will shut down too.
```

Comment:
```suggestion
Next, we let the terminal user who ran this flow exit by entering the string `q` or `quit`. When that happens, we send the `greeter` flow an exit signal so it will shut down too.
```